src/adcm_pytest_plugin/exceptions/bundles.py

```diff
@@ -24,7 +24,7 @@ def raise_if_suitable(cls, message):
         """Raise suitable bundle error if possible"""
         __tracebackhide__ = True  # pylint: disable=unused-variable
 
-        if "Wrong context" in message or "AnsibleFilterError" in message:
+        if any(map(lambda x: x in message, ["Wrong context", "AnsibleFilterError", "ERROR! the role"])):
```

Comment:
```suggestion
        if any(map(lambda x: x in message, ["Wrong context", "AnsibleFilterError", "ERROR!"])):
```

Reply:
"The role" here is just a special case here. Are you sure that we should include it here?

Reply:
fixed